### ISSUE TITLE
Advice on how to contribute, Darklaf theme broken in JMeter 5.3, 5.4, 5.5

### ISSUE THREAD

**Issue by briantully** [weisJ/darklaf#328]
**Describe the bug**
Since JMeter 5.3, JMeter switched default dark theme from Darcula to Darklaf. However Darklaf theme is unusable when using View Results Tree listener to analyze request responses, as most of the text areas that display the sampler result (Request Body, Request Headers, Response headers, etc) display text as black text on a dark background, making it unreadable. It seems that the default text foreground color should be reversed (made white) for these fields if one of the dark Darklaf themes is chosed (Darklaf, One Dark, Solaraized Dark). 

For other fields within View Results Tree (Sampler Result and Response Body), the text is correctly reversed to show white text on a dark background. These look to be textareas with text/plain formatting and do not show line numbers or monospaced font.

For the fields with issues (Request Body, Request Headers, Response headers), the textareas looks to be using a different component that contains line numbers and monospaced font, and looks to be using some code syntax component. However the syntax coloring seems to be overriding the Darklaf theme handling, and the primary text foregreound color is black instead of white -- the result being that black text is displayed on a dark background, making it unreadable.

**To Reproduce**
In JMeter 5.5, create a simple thread group with at least one HTTP Request Sampler.
Add a View Results Tree listener to the test plan.
Run the test and view the results in the View Results Tree listener.
In the left pane of View Results Tree, select "Text" as the filter (it is the default).
In the left pane of View Results Tree, select one of the sampler results, and in the right pane click on the "Request" tab. 
In the right pane of View Results Tree, select the "Request Body" sub-tab to view the request body. Note that the text is unreadable because of the black text on a dark background.
In the right pane of View Results Tree, select the "Request Headers" sub-tab to view the request headers. Note that the text is unreadable because of the black text on a dark background.
In the right pane of View Results Tree, select the "Response data" tab.
In the right pane of View Results Tree, select the "Response headers" sub-tab to view the response headers. Note that the text is unreadable because of the black text on a dark background.

**Screenshots**
https://share.cleanshot.com/dQKVNz
https://share.cleanshot.com/ajdLmW
https://share.cleanshot.com/84TIFl

**Additional Information:**
 - OS: macOS
 - OS Version: 12.5.1
- Darklaf Version: 2.7.3 (JMeter 5.5)

**Additional context**
I've been trying to figure out how to communicate this issue for a couple of years now, mostly by filing bugs against JMeter in apache jmeter bugzilla. But it hasn't gained any traction. Not sure if it's because not many Mac users use JMeter or if people just use JMeter with the default light theme. I'd like to get to the root of the issue and help fix it so that dark theme lovers like myself can use JMeter without hurting our eyes.
While I am a developer, Java is not one of my languages, so while I'd like to help contribute and fix this issue, I probably will need some hand holding. But I am motivated to learn and help contribute. I'm still using JMeter 5.2.1 as it's the last stable version that doesn't have this issue.
Thanks for reading and please let me know what I can do to help :)

**Comment by weisJ** [comment]
I think this was fixed on their side with https://github.com/apache/jmeter/commit/b914b713640ffd88b935786d13c0acce51b04259

**Comment by briantully** [comment]
> I think this was fixed on their side with [apache/jmeter@b914b71](https://github.com/apache/jmeter/commit/b914b713640ffd88b935786d13c0acce51b04259)

That would be amazing if true. Any idea how I would test that to make sure it is fixed? I assume I'd have to download the source code and build it myself with Gradle?

**Comment by weisJ** [comment]
JMeter provides nightly snapshot builds which you can try. https://jmeter.apache.org/nightly.html

**Comment by briantully** [comment]
> JMeter provides nightly snapshot builds which you can try. [jmeter.apache.org/nightly.html](https://jmeter.apache.org/nightly.html)

You are the best! Thanks so much! I downloaded the lates trunk build and tested it out and indeed the issue is resolved. Finally! Thanks so much for your help and for all you do :)

The only remaining issue that is somewhat related is making sure the "Response Body" textarea within the View Results Tree uses the same "rsyntaxtextarea" class rather than what appears to be plain text. I'll check the JMeter bug tracker to make sure the ticket I filed a couple of years ago is still listed.

Thanks again!
